plugins/github-pull-requests-board/src/components/Card/CardHeader.tsx

```diff
@@ -92,7 +92,9 @@ const CardHeader: FunctionComponent<Props> = (props: Props) => {
         <Box display="flex" alignItems="center" flexWrap="wrap" paddingTop={1}>
           <Typography variant="body2" component="p">
             Commit Status:{' '}
-            <strong>{status.commit.statusCheckRollup?.state || 'N/A'}</strong>
+            <strong>
+              {status[0].commit.statusCheckRollup?.state || 'N/A'}
```

Comment:
since the array might be empty, better to be extra carefull

```suggestion
              {status[0]?.commit.statusCheckRollup?.state || 'N/A'}
```